lib/gitlab/ci/templates/pipeline/HelmPipeline.yml

```diff
@@ -18,3 +18,19 @@ stages:
 mega-linter:
   variables:
     ENABLE: ANSIBLE
+
+# Save report from secret_detection job
+secret_detection:
+  artifacts:
+    reports:
+      secret_detection: gl-secret-detection-report.json
+    paths:
+      - gl-secret-detection-report.json
+
+# Save report from semgrep-sast scan job
```

Comment:
Can you delete this whole `semgrep-sast` code block? Also replace `- template: Jobs/SAST.gitlab-ci.yml` with `- template: Jobs/SAST-IAC.gitlab-ci.yml`? Turns out SAST.gitlab-ci.yml doesn't support Helm, but SAST-IAC does.gitlab-ci.yml

Reply:
Is there a difference between the templates at `Security/SAST-IaC.gitlab-ci.yml` and `Jobs/SAST-IaC.gitlab-ci.yml`? I added the `kics-iac-sast` block to pipelines that use either

Reply:
Looks like the Security template redirects to the Jobs template, so we should be good.

Reply:
Excellent, nice find.